frontend/src/metabase/.eslintrc

```diff
@@ -34,16 +39,16 @@
     {
       "files": ["lib/redux/hooks.ts"],
       "rules": {
-        "no-restricted-imports": [
-          "error",
-          {
-            "patterns": [
-              "metabase-enterprise",
-              "metabase-enterprise/*",
-              "cljs/metabase.lib*"
-            ]
-          }
-        ]
+        // "no-restricted-imports": [
```

Comment:
Overall looks great. Just had a house keeping question about whether or not this should stay here commented out or just be removed.